src/Dropdown/Dropdown.jsx

```diff
@@ -171,6 +177,70 @@ const Dropdown = ({
     [disabled, error, isCollapsed]
   )
 
+  const removeCurrentKeyboardFocus = useCallback(() => {
+    setCurrentKeyboardFocus(-1)
+  }, [])
+
+  const handleDropdownKeyDown = useCallback(

```

Comment:
Why not only `handleKeyDown`? We're in the Dropdown context/file.

Reply:
eb1464625c16f68cd887b40b4075fef822e0679e